images/php/fpm/Dockerfile

```diff
@@ -36,9 +37,18 @@ COPY ssmtp.conf /etc/ssmtp/ssmtp.conf
 
 ENV NEWRELIC_VERSION=8.6.0.238
 
-RUN apk del --no-cache curl libcurl \
-    && apk add --no-cache "curl=7.61.1-r2" "libcurl=7.61.1-r2" --repository http://dl-cdn.alpinelinux.org/alpine/v3.8/main/ \
-    && apk add --no-cache fcgi \
+RUN apk del --no-cache curl libcurl
```

Comment:
maybe we run the `apk del` only for php 5 and leave it installed for the other ones?